src/main/java/net/openhft/chronicle/wire/Wires.java

```diff
@@ -588,6 +588,9 @@ public static <E> E objectDate(ValueIn in, @Nullable E using) {
 
     @Nullable
     public static <E> E object0(ValueIn in, @Nullable E using, @Nullable Class clazz) {
+        if (using instanceof AbstractMarshallableCfg)
```

Comment:
We already have `Marshallable#reset` - isn't this being called? https://github.com/OpenHFT/Chronicle-Wire#a-note-on-wires-reset